util/gh-pages/index.html

```diff
@@ -240,7 +307,7 @@
 
     </style>
 </head>
-<body>
+<body ng-app="clippy" ng-controller="lintList" ng-click="toggleDropdown(undefined, $event)">
     <div id="theme-icon" class="theme-icon">&#128396;</div>
     <ul id="theme-menu" class="theme-choice" style="display: none;">
```

Comment:
The popup menus you created look a bit better than the current theme menu and they also close when the user interacts with the page body besides them. Would you mind updating the theme menu as well as part of this PR?

Reply:
I updated theme menu. Now it should close when interacting with the page